app/src/store/actions/charactersActions.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios'
+
+const DISPLAY_CARDS = 'DISPLAY_CARDS'
+
+const fetchCharacters = (dispatch) => {
+  dispatch({ DISPLAY_CARDS })
+  axios.get('https://api.jikan.moe/v3/manga/11/characters')
+    .then(res => {
+      console.log(res)
+    })
+}
+
+const thunk = (store) => (next) => (action) => {
+  if (typeof action === "object") {
+    next(action);
+  } else if (typeof action === "function") {
+    action(store.dispatch);
```

Comment:
This is a walkthrough of how the thunks work correct?

`const thunk = (store) => (next) => (action) => {
  if (typeof action === "object") {
    next(action);
  } else if (typeof action === "function") {
    action(store.dispatch);`

your actionCreator 'fetchCharacters( )' above '🔝  is an example of a thunk implementation